main.go

```diff
@@ -67,6 +67,12 @@ func main() {
 			Usage:       "Specify which ip address RDNS should generate record for",
 			Destination: &config.ConfigController.IPAddresses,
 		},
+		cli.StringFlag{
+			Name:        "features",
+			Usage:       "Specify which feature must be turned on",
```

Comment:
This is to add a backdoor in controller binary arguments to fix tests. Since we disable build and autoscaling by default, It will break CI. So by default if you are running in binary mode we enable build and autoscaling.